release.sh

```diff
@@ -5,11 +5,8 @@
 set -x
 
 # Basic tests
-check-manifest || exit 1
-isort || exit 1
-flake8 || exit 1
-py.test || exit 1
-isort -c . || exit 1
+pre-commit run --all-files || exit 1
+tox -e py39 || exit 1
```

Comment:
Moving to using GA only for this probably deserves its own PR, thought it was good idea to only make this change for now.

Reply:
:+1: